src/app/core/services/authorization.service.ts

```diff
@@ -45,7 +58,7 @@ export class AuthorizationService {
   getUserInfo(): Observable<User> {
     const token = localStorage.getItem('accessToken');
     return this.http
-      .post<User>(`${this.apiUrl}/userinfo`, token, this.httpOptions)
+      .post<User>(`${this.apiUrl}/userinfo`, { token }, this.httpOptions)
       .pipe(
```

Comment:
Add `tap` operator and make and call next method upon result:

`this.userData$$.next(user)`

Reply:
done